src/util/daysBetween.js

```diff
@@ -0,0 +1,9 @@
+function daysBetween (firstDate, secondDate) {
+  const oneDayInMilliseconds = 24 * 60 * 60 * 1000
+  const firstTime = firstDate.getTime()
+  const secondTime = secondDate.getTime()
+  const millisecondsBetween = Math.abs(firstTime - secondTime)
+  return Math.round(millisecondsBetween / oneDayInMilliseconds) + 1
+}
+
+module.exports = daysBetween
```

Comment:
Add specific tests around the daysBetween function - decided it should be an inclusive number. Used Math.round to nudge around the daylight savings cases that happen in March and October.